composer.json

```diff
@@ -35,7 +35,7 @@
         "doctrine/orm": "~2.3",
         "symfony/yaml": "~2.2",
         "symfony/validator": "~2.2",
-        "twig/twig": "~1",
+        "twig/twig": "~1.4",
```

Comment:
technically, we need at least 1.10 as we use the namespaced notation when registering the profiler template in the service tag